Makefile

```diff
@@ -5,7 +5,7 @@ BIN = selftest test
 
 all: $(BIN)
 
-utcp.o: utcp.c utcp.h utcp_priv.h
+utcp.o: utcp.c compat.h utcp.h utcp_priv.h
```

Comment:
Eigentlich müsste das auch gehen ohne `compat.h` zu erwähnen, da es ja in den .c-Dateien included wird.